doc/fbchkdoc/fmtcode.bas

```diff
@@ -145,10 +145,9 @@ function FormatFbCodeLoadKeywords( byref filename as string ) as boolean
 		'' Load the keywords, this will allow keywords to be cased.
 
 		if( filename > "" ) then
```

Comment:
Currently the logic seems broken: `if filename > ""` should be `if filename = ""`

Reply:
Yes, you are correct.  I'm so sorry.  That was very careless of me.  My test method was poor (false success), and I didn't test all cases.  

Personal observation about my own code, having not looked at it in years: these doc utility programs have a lot of duplicated start-up code, and the code is too complex for what it accomplishes.  That is an area for improvement for next time.  Otherwise, I am finding the utilities are still relevant for maintaining the wiki, even though they may not be most user friendly.

Reply:
Yea, fbdoc & co have been crucial and useful for me too for working on FB in the past years. I had wanted to make some improvements to them here and there, but never got around to completing any. It's nice that they worked generally without problems ever since I started.